PEP 1: Update outdated language referring to Travis CI and plain-text PEPs (#2347)
python/peps: pep-0001.txt

    editors do not consider whether they seem likely to be accepted.
  * The title accurately describes the content.
  * The PEP's language (spelling, grammar, sentence structure, etc.)
    and code style (examples should match :pep:`7` & :pep:`8`) should be

>     correct and conformant.  The PEP will be checked for reStructuredText
>     formatting by the repo's CIs, and will not be approved until this passes.

Reviewer: Whilst CI is a term of art, PEP authors who are primarily scientists, typing specialists, etc may not be as familiar with it -- this wording attempts to focus on the what, rather than the how.

A

```suggestion
    correct and conformant.  The PEP text will be automatically checked for 
    correct reStructuredText formatting when the pull request is submitted. 
    PEPs with invalid reST markup will not be approved.
```

=========================

PEPs are UTF-8 encoded text files using the reStructuredText_ format.

> ReStructuredText allows for rich markup that is still quite easy to

Reviewer 2: ```suggestion
reStructuredText allows for rich markup that is still quite easy to
```
https://docutils.sourceforge.io/rst.html styles it as lowercase even at the beginning of a sentence.

* Skim the PEP for obvious defects in language (spelling, grammar,
  sentence structure, etc.), and code style (examples should conform to
  :pep:`7` & :pep:`8`).  Editors may correct problems themselves, but are

>   not required to do so (reStructuredText syntax is checked by the repo's CIs).

Reviewer 3: Minor, non-blocking suggestion:
```suggestion
  not required to do so (reStructuredText syntax is checked by the repo's CI).
```
